Sources/TuistCore/Graph/Mappers/MapperEnvironment.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public protocol MapperEnvironmentKey<Value>: Hashable {
+    associatedtype Value
+    static var defaultValue: Self.Value { get }
+}
+
+/// An environment dictionary that holds extra context in mappers
+public struct MapperEnvironment {
```

Comment:
What about making this `Sendable` ensure the internal `environment` variable is accessed through a lock?

Reply:
I don't follow why we need to make this `Sendable`. It's a struct passed around, not a single object.